.buildkite/release.sh

```diff
@@ -35,5 +35,4 @@ tox -e sonar-release | grep -v "sonar.login"
 # Upload yukon_backend to PyPi
 tox -e pypi-upload | grep -v "twine upload"
 # Publish yukon_frontend to npm
-tox -e npm-publish | grep -v "//registry.npmjs.org/:_authToken="
-
+# tox -e npm-publish | grep -v "//registry.npmjs.org/:_authToken="
```

Comment:
Why is this commented out?

Reply:
It should not be run on every commit in master branch. Otherwise it fails CI.

Reply:
Yes it does. The things is that npm upload is currently broken because it's not detecting the token. But on every PR you are supposed to bump the frontend and backend versions so to do a sync release. If it is the minor or the patch versions depends on the level of change. This is in the CONTRIBUTING document.
I accepted this change because as I said, npm upload is not working correctly.

Reply:
@TSC21 I think good practice here is to use semver and release only on git tags. I can't imagine situation, when there is need to release on every commit. Also, release on every commit will require automatic version bump, and this is very bad practice, that can lead to errors.

Reply:
This will be automated eventually with automatic versioning, git tagging and release notes. This is not something you should worry about at this stage.